qiita_pet/templates/upload.html

```diff
@@ -8,9 +8,13 @@
       <br/><br/>
       Currently we can process (<b>{{extensions.replace(',',', ')}}</b>):
       <ul>
-        <li>fastq or fastq.gz (gzipped fastq) for sequences. Note that '.zip' files can not be processed.</li>
-        <li>Note that '.fasta' and '.fna' files require '.qual' files for submission </li>
-        <li>tab separated text files for sample and prep templates, the extension should be txt.</li>
+        <li>.fastq, .fastq.gz (gzipped fastq), .fna with .qual, .fasta with .qual, ,sff for sequences. </li/
```

Comment:
Can you return the {{extensions.replace(',',', ')}}, this is the full list  of extensions accepted by the system and it will depend on deployment; for example difference between main qiita release and ihmp ...

Reply:
Currently we can process (<b>{{extensions.replace(',',', ')}}</b>):
      <ul>
        <li>Note that '.zip' files can not be processed.</li>
        <li>Note that '.fasta' and '.fna' files require '.qual' files for submission. </li>
        <li>Note that '.txt. files must be tab separated and require the extension .txt. </li>
      </ul>
Because we can do more than .fastq fastq.gz and .txt now